### PR TITLE
Add support for replacing files by tiger specific versions

### PR DESCRIPTION
**Pull request by no23reason**
This allows us to use tiger specific versions of some files without some complicated replace patterns.

This support was then used to provide tiger-specific Welcome page.

JIRA: RAIL-2407

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/127/ : SUCCESS in 49s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-oss-check/106/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/128/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-oss-check/107/ : SUCCESS in 3s

**Comment by no23reason** [comment]
This is what tiger Welcome page looks like now:
localhost_3000_welcome

File: bootstrap/src/routes/Welcome--tiger.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            <h2>Your new GoodData-powered app is ready!</h2>
+            <p>
+                Now, let’s take one more step and set up your home dashboard with a test headline report
+                widget. This will help verify that everything is set up correctly.
+            </p>
+
+            <ol>
+                <li>
+                    <p>
+                        In <Code>/src/constants.js</Code>, check that <Code>backend</Code> is set to your
```
Comment (michalvanco) [inline bootstrap/src/routes/Welcome--tiger.js:39]:
```
In <Code>/src/constants.js</Code>, check that <Code>backend</Code> is set to your
GoodData Anywhere endpoint. For example, <Code>https://analytics.anywhere.gooddata.com</Code>.
```

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/129/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-oss-check/108/ : SUCCESS in 3s

**Comment by no23reason** [comment]
Updated screenshot
localhost_3000_welcome (1)

File: bootstrap/src/routes/Welcome--tiger.js (hunk 1 above)
Comment (no23reason) [reply bootstrap/src/routes/Welcome--tiger.js:39]: Updated, thank you

**Comment by lupko** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-merge-compliance-check/46/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-create-gooddata-react-app-unit-tests-zuul-docker-el7/130/ : SUCCESS in 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-create-gooddata-react-app-fossa-scanning-zuul-el7/11/ : SUCCESS in 2m 50s (non-voting)

**Pull request merged**